CONTRIBUTING.md

```diff
@@ -276,12 +276,24 @@ Like any good open source project, we use Pull Requests (PRs) to track code chan
       or explicitly request another OWNER do that for them.
     - If the owner of a PR is _not_ listed in `OWNERS`, any core maintainer may merge the PR.
 
-#### Documentation PRs
+### Documentation PRs
 
 Documentation PRs should be made on the docs repo: <https://github.com/helm/helm-www>. Keeping Helm's documentation up to date is highly desirable, and is recommended for all user facing changes. Accurate and helpful documentation is critical for effectively communicating Helm's behavior to a wide audience.
 
 Small, ad-hoc changes/PRs to Helm which introduce user facing changes, which would benefit from documentation changes, should apply the `docs needed` label. Larger changes associated with a HIP should track docs via that HIP. The `docs needed` label doesn't block PRs, and maintainers/PR reviewers should apply discretion judging in whether the `docs needed` label should be applied.
 
+### Testing PRs
```

Comment:
```suggestion
### Profiling PRs
```
I feel like you could focus this section on profiling and drop the first two sentences from the paragraph.